Source/Analytics/Read/CaseReports/GetCaseReportsPerRegionLast7Days.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq;
+using Dolittle.Queries;
+using Dolittle.ReadModels;
+using Concepts;
+
+namespace Read.CaseReports
+{
+    public class GetCaseReportsPerRegionLast7Days : IQueryFor<CaseReportsPerRegionLast7Days>
+    {
+        readonly IReadModelRepositoryFor<CaseReportsPerRegionLast7Days> _repositoryForCaseReportsPerRegionLast7Days;
+
+        public GetCaseReportsPerRegionLast7Days(IReadModelRepositoryFor<CaseReportsPerRegionLast7Days> repositoryForCaseReportsPerRegionLast7Days)
+        {
+            _repositoryForCaseReportsPerRegionLast7Days = repositoryForCaseReportsPerRegionLast7Days;
+        }
+
+        public IQueryable<CaseReportsPerRegionLast7Days> Query => 
+            _repositoryForCaseReportsPerRegionLast7Days
+                .Query
+                .Where(report => report.Id == 17925); //Testing purposes. Not using Day.Today because we have no fresh data in the database
```

Comment:
I agree, although it will make Analytics load very little data on run  – people should in some way learn why this is the case. 
Perhaps a comment with the information can be of help?

```suggestion
                .Where(report => report.Id == Day.Today); //Current testing data is static and not timestamped with today. To populate database for testing purposes, you can use a Day such as 17925.
```